_data/advisories.yaml

```diff
@@ -12,7 +12,7 @@ texts:
   rc: "The Swarm mode feature included in Docker Engine 1.12 is a release candidate feature and might be subject to non backward-compatible changes. Some functionality may change before the feature becomes generally available."
   docker4mac-beta: "Docker Desktop for Mac is currently in public beta. Some functionality may change before the product becomes generally available."
   docker4win-beta: "Docker Desktop for Windows is currently in public beta. Some functionality may change before the product becomes generally available."
-  toolbox: "**Legacy desktop solution.** Docker Toolbox is for older Mac and Windows systems that do not meet the requirements of [Docker Desktop for Mac](/docker-for-mac/) and [Docker Desktop for Windows](/docker-for-windows/). We recommend updating to the newer applications, if possible."
+  toolbox: "**Deprecated:** Docker Toolbox has been deprecated and is no longer in active development. Please use Docker Desktop instead. See [Docker Desktop for Mac](/docker-for-mac/) and [Docker Desktop for Windows](/docker-for-windows/)."
```

Comment:
If we use the ` |` approach, it's not needed to use quotes, and we could format the advisory to be a "warning" note;

```suggestion
  toolbox: |
    > **Deprecated**
    >
    > Docker Toolbox has been deprecated and is no longer in active development.
    > Please use Docker Desktop instead. See [Docker Desktop for Mac](/docker-for-mac/)
    > and [Docker Desktop for Windows](/docker-for-windows/).
    {: .warning}
```

Reply:
Oh, hmm... that doesn't work currently (something to fix)

<img width="994" alt="Screenshot 2020-10-20 at 19 45 44" src="https://user-images.githubusercontent.com/1804568/96624363-ec4df780-130c-11eb-9682-ec8dcf66d96c.png">



Oh, actually; if we remove the pages, we don't need this advisory anymore, so you can remove it

Reply:
Yes, that's correct. We don't need the toolbox advisory anymore.